core/control-panel/api/spec.did

```diff
@@ -310,8 +319,254 @@ type GetArtifactResult = variant {
   Err : ApiError;
 };
 
+// A metadata record that contains a key and a value.
+type Metadata = record {
+  key : text;
+  value : text;
+};
+
+// An entry record, which contains information and the value stored in the registry.
+type RegistryEntry = record {
+  // The UUID that identifies the entry in the registry.
+  id : UUID;
+  // The name of the entry, which is used to identify it (e.g. station). Names that start with `@` are considered
+  // to be namespaced, and the namespace is the part of the name that comes before the `/`. Within each namespace
+  // the name should refer to the same type of entry, but many entries can exist with the same name.
+  //
+  // e.g. if the namespace is "@orbit" and the name is "station", then all the entries will refer to a wasm module.
+  //
+  // Restrictions:
+  //
+  // - Names that start with `@` are considered namespaced.
+  // - Names that start with `@` must have a namespace and a name separated by a `/`.
+  // - Names must be between 2 and 48 characters long.
+  // - Namespaces must be between 2 and 32 characters long.
+  // - Names that are not namespaced, are put in the default namespace `@a`.
```

Comment:
default namespace is `@default`, same for the comment in the Rust counterpart

Reply:
good catch